tests/utils/lambda-helpers.mjs

```diff
@@ -143,7 +143,7 @@ export async function loadFunction(ctx, { env } = {}) {
         flags: flags ?? DEFAULT_FLAGS,
       },
       lambdaFunc: { handler },
-      timeoutMs: 4_000,
+      timeoutMs: 10_000,
```

Comment:
bumped function invocation timeout to allow for 5s sleep in added test scenarios